File: Solo Learn/Médio/Isso é Ímpar/thats_odd.py
# Function to find the sum of all even integers in a list
def sum_of_evens(lst):
    return sum(num for num in lst if num % 2 == 0)

# Read the length of the list
N = int(input())

# Read the list elements
numbers = [int(input()) for _ in range(N)]

# Calculate the sum of even numbers
result = sum_of_evens(numbers)

# Print the result
print(result)

File: Solo Learn/Médio/Isso é Ímpar/thats_odd.c
#include <stdio.h>

int main() {
    int N, num, sum = 0;

    // Read the length of the list
    scanf("%d", &N);

    // Iterate through the list elements
    for (int i = 0; i < N; i++) {
        // Read each element
        scanf("%d", &num);

        // Check if the number is even
        if (num % 2 == 0) {
            // Add to the sum if even
            sum += num;
        }
    }

    // Print the result
    printf("%d\n", sum);

    return 0;
}
